CITATION.cff

```diff
@@ -0,0 +1,32 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
```

Comment:
Use it at all, or use it in academic contexts where a citation is appropriate? The Python docs won't be citing Sphinx...
```suggestion
message: "If you use Sphinx, please cite it as below."
```

Reply:
I think this is obvious from context. Github currently ignores this message and uses its own, though.